config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: controller
-  newTag: latest
+  newName: bpalmer/oria-operator
+  newTag: test
```

Comment:
Do we really want to commit `bpalmer` here?

Reply:
Oops, was using that as my IMG env variable when running `make docker-build`. Will have to make sure to revert that. Good catch!